gems/sorbet-runtime/lib/types/enum.rb

```diff
@@ -88,7 +88,7 @@ def self.from_serialized(serialized_val)
 
   # Note: It would have been nice to make this method final before people started overriding it.
   # @return [Boolean] Does the given serialized value correspond with any of this enum's values.
-  sig {overridable.params(serialized_val: SerializedVal).returns(T::Boolean)}
+  sig {overridable.params(serialized_val: SerializedVal).returns(T::Boolean).checked(:never)}
```

Comment:
We could also change `serialized_val: SerializedVal` to `serialized_val: T.untyped` instead?
WDYT?

Reply:
`SerializedVal` is already a type alias for `T.untyped`.

It's a type alias to make it easier for people changing the T::Enum code which arguments / returns are meant to be the serialized values (it was confusing before when it had `T.untyped` in line, because it made the types look like someone was being lazy, but there's a good reason why we can't add a type here.